src/test/java/org/broadinstitute/consent/http/resources/ElectionResourceTest.java

```diff
@@ -125,6 +125,20 @@ public void testIsDataSetElectionOpenError() {
         Assert.assertEquals(ERROR, response.getStatus());
     }
 
+    @Test
+    public void testDescribeVotesOnElection() {
+        Response response = electionResource.describeVotesOnElection(randomInt());
+        Assert.assertEquals(OK, response.getStatus());
+    }
+
+    @Test
+    public void testDescribeVotesOnElectionError() {
+        when(voteService.findVotesByElectionId(randomInt())).thenThrow(new NotFoundException());
+        electionResource = new ElectionResource(voteService, electionService);
+        Response response = electionResource.describeVotesOnElection(randomInt());
+        Assert.assertEquals(NOT_FOUND, response.getStatus());
+    }
```

Comment:
currently testDescribeVotesOnElectionError does not work, as it still returns a 200 response status, I'm not really sure why, thoughts?

Reply:
Yes ... the issue is with the random int ...
```
        when(voteService.findVotesByElectionId(randomInt())).thenThrow(new NotFoundException());
        ...
        Response response = electionResource.describeVotesOnElection(randomInt());
```
Note that `randomInt()` returns an integer. It is almost guaranteed that the two times you are using it are returning different integers, so the when condition is never triggered. Try changing the when to
```
        when(voteService.findVotesByElectionId(any())).thenThrow(new NotFoundException());
```

Reply:
ah makes sense thank you!